hrm-service/unit-tests/contact/contact.test.ts

```diff
@@ -0,0 +1,384 @@
+import * as contactDb from '../../src/contact/contact-data-access';
+import {
+  connectContactToCase,
+  createContact,
+  CSAMReportEntry,
+  patchContact,
+  SearchContact,
+  searchContacts,
+} from '../../src/contact/contact';
+import { ContactBuilder } from './contact-builder';
+import { omit } from 'lodash';
+
+jest.mock('../../src/contact/contact-data-access');
+
+const mockContact: contactDb.Contact = {
+  id: 1234,
+  accountSid: 'accountSid',
+  csamReports: [],
+};
+
+describe('createContact', () => {
+  const sampleCreateContactPayload = {
+    rawJson: {
+      childInformation: {
+        name: {
+          firstName: 'Lorna',
+          lastName: 'Ballantyne',
+        },
+      },
+      callType: 'carrier pigeon',
+      caseInformation: {
+        categories: {
+          a: {
+            category: true,
+          },
+        },
+      },
+    },
+    queueName: 'Q',
+    conversationDuration: 100,
+    twilioWorkerId: 'owning-worker-id',
+    timeOfContact: new Date(2010, 5, 15),
+    createdBy: 'ignored-worker-id',
+    helpline: 'a helpline',
+    taskId: 'a task',
+    channel: 'morse code',
+    number: "that's numberwang",
```

Comment:
This is my favorite PR so far xD

Reply:
Do you get that reference? I didn't think that show was known much outside the UK 😂